packages/components/src/components/dropdown/dropdown.scss

```diff
@@ -30,7 +30,6 @@
         &:active,
         &:focus {
             outline: 0;
-            border-color: var(--border-active);
```

Comment:
I deleted this style property because it shouldn't apply to active input. It should apply only when dropdown is active, not only when input is on focus(applied in packages/components/src/components/dropdown/dropdown.scss on line 242)

Reply:
@kate-deriv make sure that QA is testing all the dropdown in light and dark mode